src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/testhosts/TcpDefaultResourceTestServiceHost.cs

```diff
@@ -16,11 +16,11 @@ protected override ServiceHost CreateServiceHost(Type serviceType, Uri[] baseAdd
     }
```

Comment:
I can't add a comment on the correct line, but the copyright notice needs to be updated to the one standard one we use for .NET Core projects